src/schemas/json/template.json

```diff
@@ -6,6 +6,13 @@
   "required": [ "author", "classifications", "identity", "name", "shortName" ],
 
   "definitions": {
+    "datatype": {
+      "description": "Indicates limitations on the valid values a symbol may be assigned. At this point, the only valid datatype is \"choice\", which also requires providing symbols.choices configuration for the symbol.",
```

Comment:
I think the wording here needs some work, particularly: `...the only valid datatype is \"choice\"...`. All of the values listed under "anyOf" are valid datatypes. Is this trying to get across the idea that "choice" is the only datatype that can be configured to take on explicitly defined values? Perhaps the second sentence should be removed entirely.